security/acme-client/src/opnsense/mvc/app/models/OPNsense/AcmeClient/AcmeClient.xml

```diff
@@ -366,6 +366,7 @@
                         <dns_azure>Azure DNS API</dns_azure>
                         <dns_cf>CloudFlare.com API</dns_cf>
                         <dns_cloudns>ClouDNS API</dns_cloudns>
+                        <dns_cn>Core-Networks API</dns_cn>
```

Comment:
Would prefer to see this sorted by API name (instead of dns_* XML tag).

Reply:
done